src/main/java/com/bigbookmarket/domain/Message.java

```diff
@@ -0,0 +1,32 @@
+package com.bigbookmarket.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Message extends BaseTimeEntity {
+
+    @Id
+    @Column(columnDefinition = "BIGINT")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long messageId;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "senderId", referencedColumnName = "userId")
+    private User user;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "receiverId", referencedColumnName = "userId")
+    private User user2;
```

Comment:
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "senderId", referencedColumnName = "userId")
    private User sender;

    @ManyToOne(fetch = FetchType.LAZY)
    @JoinColumn(name = "receiverId", referencedColumnName = "userId")
    private User receiver;
```

이럴 때는 변수명을 sender or senderUser, receiver or receiverUser라고 하는게 안 헷갈리지 않을까요?
쪽지부분 구현은 민영님께 부탁드릴 예정인데 민영님께서 user, user2가 편하시면 이렇게 하셔도 됩니다~